src/main.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+function daysInMonth(month, year) {
+  if (Number.isInteger(month) && Number.isInteger(year) && month < 13 && month !== 0) {
+    let daysInMon = new Date(year, month, 0).getDate();
+      console.log(daysInMon)
```

Comment:
it must `return number` instead of `return undefined`

Reply:
it returns the number of days.